client/vzRsFiltAtt_service.go

```diff
@@ -0,0 +1,48 @@
+package client
+
+import (
+	"fmt"
+
+	"github.com/ciscoecosystem/aci-go-client/models"
+)
+
+func (sm *ServiceManager) CreateFilterRelationship(tnVzFilterName string, contract_subject string, contract string, tenant string, vzRsFiltAttAttr models.FilterRelationshipAttributes) (*models.FilterRelationship, error) {
+	rn := fmt.Sprintf(models.RnvzRsFiltAtt, tnVzFilterName)
+	parentDn := fmt.Sprintf(models.ParentDnvzRsFiltAtt, tenant, contract, contract_subject)
```

Comment:
we should get parentDn directly from the signature and not have tenant, contract, subject there.